themes/tiddlywiki/vanilla/base.tid

```diff
@@ -627,6 +627,10 @@ button.tc-untagged-label {
 
 .tc-page-controls {
 	margin-top: 14px;
+}
+
+.tc-page-controls > span > button,
+.tc-page-controls > span > span > button{
```

Comment:
@pmario 

> There is no button element in the language switcher dropdown.

It's not about a button in the language switcher dropdown. It's about addressing solely the button contents in the pagecontrols, but not any popup reveals.

The point is specifically that I only want to address the buttons, not everything in PageControls and why I changed that line to begin with.